src/Icons/doc/index.rst

```diff
@@ -237,7 +237,7 @@ site *on-demand*:
    fetched (and cached).
 
 That's all. This works by using the `Iconify`_ API (to which `ux.symfony.com/icons`_
-is a frontend for) to fetch the icon and render it in place. This icon is then cached
+is a frontend) to fetch the icon and render it in place. This icon is then cached
```

Comment:
I'm not super-fluent in english, but to me the previous version sounds better, do?

Reply:
Thanks!